### ISSUE TITLE
Loss of focus in the presence of a corette pointing at a smiley face.

### ISSUE THREAD

**Issue by TinySmallM** [ealush/emoji-picker-react#320]
I'm trying to use emoticons inside contentEditable. There I can edit the text and have a corette.
As soon as I'm in the open menu where the smiley face is, I lose focus

I tried to find props  setting that would cancel this behavior, but I couldn't figure it out.

I ask you to note that this should work with previews.

**Comment by ealush** [comment]
Hey👋
I'm not sure what exactly you're experiencing. Can you share a video of the issue?

**Comment by donnikitos** [comment]
+1 
The input looses focus whenever you start hovering over an emoji:

https://user-images.githubusercontent.com/5833538/220611559-68d954cc-5af8-453f-82ea-c278e0102374.mp4

**Comment by ealush** [comment]
This is the intended behavior, as it enhahces the keyboard navigation capabilities.

Once you hover over an emoji, or even scroll down, you can continue with keyboard navigation. Hitting esc, for example, will bring the focus back to the input.

How else would you prefer that? A prop to control this behavior, or another way to flip the default?

**Comment by donnikitos** [comment]
IMO that behavior is not fully intuitive, e.g. if you open the selector in a popup and hover an emoji you are not able to continue typing.

Have you considered using a simulated focus state?
E.g. you could store the last hovered emoji in a state and navigate via keyboard from there

**Comment by ealush** [comment]
Yes, I have, but I believe that may introduce a different set of problems.

Now, when you hit left or right, do you move the cursor, or the emojis?

One thing we may consider is changing the behavior if the input has text, but I am still not certain it would make the problem go away

**Comment by dmush** [comment]
Hey
I'm not very sure but that probably makes a bit of an issue on mobile.

The case is:
I have an input and the picker on the page (similar to the video above). When I focus on the input the keyboard appears on the screen. I type something and then click an emoji. The keyboard hides because the input looses focus. In order to prevent the keyboard from hiding I do the `input.focus()` in my script. But that doesn't always work as expected. I would say that it works sometimes. In order to workaround that I do the `.focus()` and then another `.focus()` wrapped in the `requestAnimationFrame`. And that seem to work good.

Does it makes sense to disable the keyboard navigation on mobile? Or add the prop that enables/disables the keyboard navigation (as suggested above)?

**Comment by devmalcolm** [comment]
Hey👋
I heard v4 was here, and i still was wondering if this issue still active? I re-installed the package, and it seems that the loss focus issue still appeared, after reading the doc i sadly do not find a way to control this behavior, does it planned to be fixed ? Thanks !
